server/package.json

```diff
@@ -21,7 +21,7 @@
   "scripts": {
     "migrate": "knex --knexfile=./knexfile.ts migrate:latest",
     "seed-db": "knex --knexfile=./knexfile.ts seed:run",
-    "start": "knex --knexfile=./knexfile.ts migrate:latest && knex --knexfile=./knexfile.ts seed:run && ts-node ./src/server.ts",
+    "start": "ts-node ./src/server.ts",
```

Comment:
Is there a reason to remove the migration from the start script? If not, it would be neat to include it (because I will surely forgot to run `yarn migrate` in the beginning)

Reply:
The reason I took it out was that the seed-db was in between.

Although this could be added before both seed-db and start, since there is no harm done trying to migrate multiple times.